tests/reg_issue25_unbox_outerref/unboxOuterRef.fz

```diff
@@ -34,11 +34,11 @@ unboxOuterRef is
 
   y (i i32) is
     redef as_string => "y $i"
-    g (v y) is
+    g (v ref y) is
```

Comment:
instead of `ref y`, my idea here was that this should be `y.this.type`. But then it can no longer be called as in `g (y i)`. 

Would maybe be good to have three cases here
```
  gy (v y) is...
  gry (v ref y) is...
  gyt (v y.this.type) is...
```
and then call all of these
```
  // gy y.this   -- incomptible types
  gy (y i) 
  gry y.this
  gry (y i)
  gyt y.this
  // gyt (y i) -- incompatible types
```